README.md

```diff
@@ -6,12 +6,103 @@ The `TDS_ConfigService` (aka Config Support Service) consists of two modules:
 * **service:** REST endpoints that provide TDS configuration data
 
 ## Build
-* To build the **client**:
-  * `mvn clean install -f /path/to/client/pom.xml`
-* To build the **service**:
-  * `mvn clean install -f /path/to/service/pom.xml`
-* To build the service and run integration tests:
-  * `mvn clean install -Dintegration-tests.skip=false -f /path/to/service/pom.xml`
+To build the **client**:
+
+* `mvn clean install -f /path/to/client/pom.xml`
+
+To build the **service**:
+
+* `mvn clean install -f /path/to/service/pom.xml`
+
+To build the service and run integration tests:
+  
+* `mvn clean install -Dintegration-tests.skip=false -f /path/to/service/pom.xml`
+
+To build the service and its associated Docker image:
+
+* `mvn clean install docker:build -f /path/to/service/pom.xml`
```

Comment:
Should we add any info on where to download and install what is needed for Docker?